resources/public/opensearch.xml

```diff
@@ -0,0 +1,14 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
+                       xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+    <ShortName>Cljdoc</ShortName>
+    <Description>Cljdoc is a website building &amp; hosting documentation for Clojure/Script libraries</Description>
+    <InputEncoding>UTF-8</InputEncoding>
+    <Image width="16" height="16" type="image/x-icon">https://cljdoc.org/favicon.ico</Image>
+    <Url type="text/html" template="https://cljdoc.org/search">
+        <Param name="q" value="{searchTerms}"/>
+    </Url>
+    <Url type="application/x-suggestions+json" template="https://cljdoc.org/suggest">
+        <Param name="q" value="{searchTerms}"/>
+    </Url>
```

Comment:
would it perhaps make sense to comment that out for now to avoid unnecessary requests?

Reply:
Thanks for the context, makes sense 👍